### DISCUSSION TITLE
[Scenario Editor] Graveyard sprites present in the files and not used in the original version of the game

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#8785] [General]
The graveyard sprites present in the game files and not used in the original version of the game are not of the same graphic quality as the game standard.
They certainly haven't been integrated into the game because they're so ugly.
I suggest removing them from the scenario editor because they don't bring anything new or good to the game.
It's not because there are files present and not used in the original game that they should be integrated into fheroes2.

In fheroes2:
**My eyes are bleeding.**
![What the heck  So ugly  0](https://github.com/ihhub/fheroes2/assets/43583994/89db0a95-152b-41cb-a30c-5adc8ac19d2c)

**Comment by Alucard648** [comment]
I think, one of the OG maps use this graveysrd variant.

**Reply by LeHerosInconnu** [reply]
Hello @Alucard648,

I think the scenario creator modified the scenario file or it was a bug.
It also happened and reported that one of the versions of the random map generator could place the said sprite on the adventure map.

**Reply by LeHerosInconnu** [reply]
It seems that this graphic was only used in the “Peasants” scenario from the PoL expansion.

In original scenario editor:

<img width="656" height="538" alt="Ugly graveyard 001" src="https://github.com/user-attachments/assets/71f94de1-f795-4251-9f53-75be38b85c70" />

<img width="656" height="538" alt="Ugly graveyard 002" src="https://github.com/user-attachments/assets/73b426ec-eb4c-4bd3-b7b0-783045f3e847" />
